tests/mir-opt/dataflow-const-prop/default_boxed_slice.rs

```diff
@@ -20,6 +20,6 @@ fn main() {
     // CHECK: debug a => [[a:_.*]];
     // We may check other inlined functions as well...
 
-    // CHECK: {{_.*}} = const Box::<[bool]>(
+    // CHECK: {{_.*}} = {{(const )?}}Box::<[bool]>(
```

Comment:
Having to change this is problematic, since apparently the intent of the test was to ensure that this indeed turns into a constant.

Reply:
Oh, I entirely agree with that. I feel like at this point we are rather exploring the impact of adding those contracts than producing final solutions.